src/azure-cli/azure/cli/command_modules/vm/generated/commands.py

```diff
@@ -9,24 +9,132 @@
-    # with self.command_group('vm ssh-public-key', vm_ssh_public_key, client_factory=cf_ssh_public_key) as g:
-    #     g.custom_command('list', 'vm_ssh_public_key_list')
-    #     g.custom_show_command('show', 'vm_ssh_public_key_show')
-    #     g.custom_command('create', 'vm_ssh_public_key_create')
-    #     g.custom_command('update', 'vm_ssh_public_key_update')
-    #     g.custom_command('delete', 'vm_ssh_public_key_delete', confirmation=True)
-    #     g.custom_command('generate-key-pair', 'vm_ssh_public_key_generate_key_pair')
-    #
-    # with self.command_group('vm', is_experimental=True):
-    #     pass
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+
+    from ..generated._client_factory import cf_ssh_public_key
+
+    vm_ssh_public_key = CliCommandType(
+        operations_tmpl='azure.mgmt.compute.operations._ssh_public_keys_operations#SshPublicKeysOperations.{}',
+        client_factory=cf_ssh_public_key,
+    )
+    with self.command_group('vm ssh-public-key', vm_ssh_public_key, client_factory=cf_ssh_public_key) as g:
+        g.custom_command('delete', 'vm_ssh_public_key_delete', confirmation=True)
+
+    from ..generated._client_factory import cf_virtual_machine
+
+    vm_virtual_machine = CliCommandType(
+        operations_tmpl='azure.mgmt.compute.operations._virtual_machines_operations#VirtualMachinesOperations.{}',
+        client_factory=cf_virtual_machine,
+    )
+    with self.command_group('vm virtual-machine', vm_virtual_machine, client_factory=cf_virtual_machine) as g:
+        g.custom_command('reimage', 'vm_virtual_machine_reimage')
+
+    from ..generated._client_factory import cf_virtual_machine_scale_set
+
+    vm_virtual_machine_scale_set = CliCommandType(
+        operations_tmpl=(
+            'azure.mgmt.compute.operations._virtual_machine_scale_sets_operations#VirtualMachineScaleSetsOperations.{}'
+        ),
+        client_factory=cf_virtual_machine_scale_set,
+    )
+    with self.command_group(
+        'vm virtual-machine-scale-set', vm_virtual_machine_scale_set, client_factory=cf_virtual_machine_scale_set
+    ) as g:
+        g.custom_command(
+            'force-recovery-service-fabric-platform-update-domain-walk',
+            'vm_virtual_machine_scale_set_force_recovery_service_fabric_platform_update_domain_walk',
+        )
+        g.custom_command('redeploy', 'vm_virtual_machine_scale_set_redeploy')
+        g.custom_command('reimage-all', 'vm_virtual_machine_scale_set_reimage_all')
+
+    from ..generated._client_factory import cf_virtual_machine_scale_set_vm_extension
+
+    vm_virtual_machine_scale_set_vm_extension = CliCommandType(
+        operations_tmpl='azure.mgmt.compute.operations._virtual_machine_scale_set_vm_extensions_operations#VirtualMachineScaleSetVmExtensionsOperations.{}',
+        client_factory=cf_virtual_machine_scale_set_vm_extension,
+    )
+    with self.command_group(
+        'vm virtual-machine-scale-set-vm-extension',
```

Comment:
`vmss extension` is already covered. @qwordy can you help update the resource coverage report?